src/System.Diagnostics.Process/ref/project.json

```diff
@@ -4,12 +4,14 @@
     "System.Runtime": "4.4.0-beta-24714-01",
     "System.Runtime.Extensions": "4.4.0-beta-24714-01",
     "System.Runtime.Handles": "4.4.0-beta-24714-01",
+    "System.Runtime.InteropServices": "4.4.0-beta-24714-01",
```

Comment:
The `System.Security.SecureString` was forwarded to `System.Runtime.IteropServices` so we need this dependency.